.circleci/config.yml

```diff
@@ -8,41 +8,11 @@ executors:
     docker:
       - image: thecodingmachine/php:8.0-v4-cli
 jobs:
-  install-and-self-analyse:
+  "Code Style Analysis":
     executor: php-74
     steps:
       - checkout
-      - run: date "+%F" > /tmp/cachekey; cat composer.json >> /tmp/cachekey
-      - restore_cache:
-          keys:
-            - composer-v4
-      - restore_cache:
-          keys:
-            - psalm-cache-{{ checksum "/tmp/cachekey" }} # speeds up diff run
       - run: composer update
-
-      - save_cache:
-          key: composer-v4
-          paths:
-            - /home/docker/.composer/cache/files
-            - /home/docker/.composer/cache/vcs
-
-      - run:
-          name: Static analysis
-          command: php -dextension=pcntl.so ./psalm --threads=10
-
-      - save_cache:
-          key: psalm-cache-{{ checksum "/tmp/cachekey" }}
-          paths:
-            - /tmp/psalm
```

Comment:
These `save_cache` calls were probably quite useful

Reply:
Seemed to me like they were used between builds. They're not mentioned anywhere else. Should I keep them anyway on the next job?

Reply:
I think composer cache should be used by the job that does `composer install`. It speeded up both the `composer install` (it wouldn't need to fetch most of packages afresh) and Psalm itself. Now that the self-check is not performed we can drop psalm cache, but composer cache is still useful.